docker/wiremock/stubs/mappings/retrieveDocumentResources_localhost.json

```diff
@@ -0,0 +1,26 @@
+{
+  "priority": 1,
+  "request": {
+    "method": "GET",
+    "url": "/GP0001/STU3/1/gpconnect/fhir/Patient/2/DocumentReference?_include=DocumentReference%3Asubject%3APatient&_include=DocumentReference%3Acustodian%3AOrganization&_include=DocumentReference%3Aauthor%3AOrganization&_include=DocumentReference%3Aauthor%3APractitioner&_revinclude%3Arecurse=PractitionerRole%3Apractitioner",
+    "headers": {
+      "Host": {
+        "equalTo": "localhost"
```

Comment:
This is not a workable solution. What if we need a hostname that is not wiremock or localhost? We will be deploying this into AWS for VP testing. The DNS name will be dynamically assigned then.

Reply:
We do this correctly for SDS `correctSDSResponse.json:94`

Reply:
caused by manually setting target host - header